### PR TITLE
Added design for Product Listing Page

### PR DESCRIPTION
**Pull request by shangkaul**

### CODE REVIEW

**Comment by yaswanthmyneni** [comment]
You can remove this icon as you added a button to the wishlist.
Screenshot 2022-02-14 at 12 04 26 PM

**Comment by yaswanthmyneni** [comment]
also, decrease the height of the cards for better ux. The code looks good, can be merged.

File: pages/productList/productList.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+<div class="main-body">
+    <div class="grid grid-sidemenu">
+        <aside class="grid-item grid-menu">
+            <div>
+                <h3>Price</h3>
+                <div class="check-grp">
+                <input type="checkbox" name="price" value="1000"> ₹1000 - ₹2000<br/>
+                <input type="checkbox" name="price" value="2000"> ₹2000 - ₹2500<br/>
+                <input type="checkbox" name="price" value="2500"> ₹2500 - ₹3500<br/>
+                <input type="checkbox" name="price" value="3500"> ₹3500 + <br/>
```
Comment (Naman-Saxena1) [inline pages/productList/productList.html:34]: Rather than writing like this, you should add labels for input.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Code looks fine to me, can't find any major issues in it 👍

**Comment by shangkaul** [comment]
> also, decrease the height of the cards for better ux. The code looks good, can be merged.

thanks, I'll look into this.

**Pull request merged**